src/components/Visualization/Visualization.js

```diff
@@ -64,6 +64,8 @@ import {
 export const DEFAULT_SORT_DIRECTION = 'asc'
 export const FIRST_PAGE = 1
 export const PAGE_SIZE = 100
+// min width of "Rows per page" select + 2 * padding
```

Comment:
what happens when a different language is used?

Reply:
Shoot. I hadn't considered that. I just tested this and things do break if I add text to the "Rows per page" pre-fix on the select, so well spotted. 

Since we can't know the `width` of this select upfront, the only solution would be to measure it. And this would involve adding a `ref` to that select, which we can't currently as it's part of the `Pagination` component in @dhis2/ui. However, adding a prop for it there would also be quite odd, I guess we'd have to call it `pagesizeSelectRef` and then (for consistency's sake) we would probably want to add these types of props for all the sub-components. Or alternatively/preferably, we would expose all the individual sub-components and allow them to accept a ref, and probably then we need to expose some functions which make up the internal logic of the component.

I'm not saying this is impossible. But I am wondering if such a high-effort solution would be worth it for what is effectively a bit of an edge case: this problem only occurs if the complete table has a smaller width than the "Rows per page" select.

I think I'd be in favour of keeping a hard coded value, until we receive any actual bug reports for this. Perhaps we should add a bit of a safety margin and increase it by 30% to accommodate for the fact that English is quite a concise language?